WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -1,28 +1,39 @@
 package org.wordpress.android.ui.posts
 
-import android.content.DialogInterface
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.FrameLayout
+import androidx.annotation.NonNull
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.ViewModelProviders
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.bottomsheet.BottomSheetDialog
 import org.wordpress.android.R
 import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.login.widgets.WPBottomSheetDialogFragment
+import org.wordpress.android.ui.posts.ActionState.TagsActionState
+import org.wordpress.android.ui.posts.PrepublishingScreen.HOME
 import javax.inject.Inject
 
-class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment() {
+class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(), TagsSelectedListener {
     @Inject internal lateinit var viewModelFactory: ViewModelProvider.Factory
 
-    private lateinit var viewModel: PrepublishingViewModel
+    private lateinit var prepublishingViewModel: PrepublishingViewModel
+    private lateinit var prepublishingActionsViewModel: PrepublishingActionsViewModel
+
+    private lateinit var site: SiteModel
```

Comment:
Do we need to keep the reference to the SiteModel in the fragment? What if we just passed it to the VM?

Reply:
Good idea. Thanks!